modules/component/src/componentNext.ts

```diff
@@ -81,10 +55,24 @@ export class ComponentNext<P1 extends ComponentProps> {
     private readonly _comparator: Comparator
   ) {}
 
+  /**
+   * Transforms the component from ComponentNext P1 to ComponentNext P2
```

Comment:
Use imperative voice instead of descriptive voice. So "transform Comp<P1> to Comp<P2>", rather than "it transforms..."